android/titanium/src/java/org/appcelerator/titanium/TiC.java

```diff
@@ -1598,6 +1613,11 @@
 	/**
 	 * @module.api
 	 */
+	public static final String PROPERTY_IDENTIFIER = "identifier";
```

Comment:
Using @hansemannn sample code, I think it was to identify the menu item using user's given id for said item in app.js. Normally an Android developer would have to change the menu xml file but since the user will be using JS, I was going to do it for them programmatically.